src/app/shell/personal-cabinet/parent/create-application/create-application.component.ts

```diff
@@ -99,6 +100,7 @@ export class CreateApplicationComponent implements OnInit, OnDestroy {
     this.children$.pipe(filter(Boolean), takeUntil(this.destroy$)).subscribe((children: SearchResponse<Child[]>) => {
       this.parentCard = children.entities.find((child: Child) => child.isParent);
       this.children = children.entities.filter((child: Child) => !child.isParent);
+      this.has20Children = this.children.length >= 20;
```

Comment:
A magic number in the property name is not the best option.
It is better to name it something like `isChildLimitReached` and create a constant MAX_CHILD or similar